### ISSUE TITLE
scuba diving gas mixtures

### ISSUE THREAD

**Issue by Morningstar88** [Morningstar88/kalki-search#210]
https://www.google.com/search?q=guide+to+gas+mixes+for+diving&gs_ivs=1#tts=0

**Comment by Morningstar88** [comment]
The most common mixes are 32 percent and 36 percent oxygen, also known as EAN32 and EAN 36. Many people think that nitrox allows you to dive deeper and stay down longer, but this is simply not true. Nitrox is a shallow-diving gas.

**Comment by Morningstar88** [comment]
We can understand from the above discussion that deep-sea divers use a mixture of helium and oxygen. Therefore, the correct answer is option (A) helium-oxygen.14 Oct 2020

**Comment by Morningstar88** [comment]
The most commonly used gas blend for sport or recreational diving is Nitrox up to 40% oxygen. Technical divers often use Nitrox gas mixes with higher level of oxygen up to 100% to accelerate decompression. More advanced technical divers use Trimix so as to be able to dive safely to depths greater than 60 metres.

**Comment by Morningstar88** [comment]
https://www.godlikeproductions.com/forum1/message930731/pg1

**Comment by Morningstar88** [comment]
The most common mixes are 32 and 36 percent oxygen, also known as EAN32 and EAN36. Reduced nitrogen loading allows divers to stay underwater longer, however, because of the increased levels of oxygen they cannot dive as deep as they would be able to with air.

**Comment by Morningstar88** [comment]
Nitrox
Nitrox is the second most commonly used gas in diving after air. It is different from air in that it simply has higher levels of oxygen. Nitrox can contain from 22 to 40 percent oxygen. The most common mixes are 32 and 36 percent oxygen, also known as EAN32 and EAN36. Reduced nitrogen loading allows divers to stay underwater longer, however, because of the increased levels of oxygen they cannot dive as deep as they would be able to with air.

Due to such nature of this gas mix, Nitrox is most often used for repetitive diving, such as on Liveaboards, where the customers may be completing multiple dives in a day, for over a week. Nitrox can also be used to accelerate in-water decompression stops.

Remember though, diving with Nitrox requires special training! You are also advised to have your own gas analyzer to check whether your gas was blended correctly, as even a seemingly small difference in the percentage of oxygen in your mix will make a big difference in regards to the safe depth and time underwater.

**Comment by Morningstar88** [comment]
Air
As it’s already been mentioned, atmospheric air is the most common diving gas mix. It normally consists of approximately 21% oxygen, 78% nitrogen and 1% other trace gases, primarily argon. To simplify the calculations this last 1% is usually treated as if it were nitrogen. The main benefits of using air for diving are that it’s cheap and easy to work with. However, the nitrogen component of it causes nitrogen narcosis, which is why it has a safe depth limit of about 40 meters (130 feet). If you go beyond 58 meters (190 feet), where the pO2 rises to 1.4 ATA, oxygen will also become toxic, causing sensory distortions and seizures. For this reason, the maximum operating depth of air is 66.2 meters (218 feet).

But what if the divers need to go deeper or stay underwater longer than it is possible when breathing air? That’s where all the different alternative gas blends come into play.

**Comment by Morningstar88** [comment]
https://www.godlikeproductions.com/forum1/message4140182/pg1

**Comment by Morningstar88** [comment]
https://www.youtube.com/watch?v=3V9QHBgrPNY&t=56s

**Comment by Morningstar88** [comment]
Helium Beer Test | Helium Infused Beer | Short Version with English Subtitles

**Comment by Morningstar88** [comment]
Helium can be found in abundance all over the universe. In fact, it is the second most abundant element known to man. Because of its chemical composition, it is not possible to produce helium artificially, which is why it is extracted from natural gas wells.

**Comment by Morningstar88** [comment]
https://science.howstuffworks.com/question12.htm#:~:text=Helium%20can%20be%20found%20in,extracted%20from%20natural%20gas%20wells.

**Comment by Morningstar88** [comment]
Who uses the most helium?
The largest consumer of helium is NASA, which uses around 75 million cubic feet of helium. This is closely followed by the U.S. Department of Defense, which uses helium to cool oxygen and hydrogen used as rocket fuel.

**Comment by Morningstar88** [comment]
https://www.google.com/search?q=how+to+buy+helium+gas&gs_ivs=1

**Comment by Morningstar88** [comment]
https://www.quora.com/Where-can-I-buy-100-helium-I-need-it-for-a-project-I-m-working-on-however-it-needs-to-be-at-least-90-purity-or-more

**Comment by Morningstar88** [comment]
[Where can I buy helium online?](https://www.quora.com/Where-can-I-buy-helium-online)
You can buy it from Amazon and other vendors. It’s not hard to find online, but I wouldn’t recommend it. The tiny tank you get is disposable, so it’s a huge waste. There’s almost nothing in that tank, so you’d be paying quite a lot for it. And I believe the purity of the helium in those portable tanks is pretty low. All helium comes with a grade. That grade determines how pure the mix is. (You’re never going to get 100% pure helium in a cylinder.) You can read more about the grade of helium here; [Grades of helium: the differences and uses.](https://zephyrsolutions.com/what-are-the-different-grades-of-helium-and-what-are-they-used-for/)

You’re going to do a lot better (price, grade, quantity) through a gas distributor. If you’re not finding one in your area by searching for “helium” or “gas supply,” search for “welding supply.” No one is going to ship cylinders of industrial grade helium. Aside from the weight of the steel tanks used for most gas distribution, the contents are under extreme pressure and they can’t simply be packaged and shipped.

**Comment by Morningstar88** [comment]
Helium has many uses, from cooling MRI machines to finding leaks in ships, but there are many different grades of helium. Which is which? We break it down.

Helium is a little more complex than most people think—it’s not just for party balloons. Helium is needed for welding, microscopes, airbags, ship inspection, computers, TVs, smart phones, MRIs, blimps, space exploration, diving, meterology, scientific research, the Internet… [the list goes on and on](http://askzephyr.com/15-uses-for-helium-you-never-knew/).

The point is, different grades of helium are needed for many of these different applications and industries. Without getting too nerdy about it, let’s break down some of the most common grades of helium.

 

How we measure grades of helium
Before we get into the different grades of helium, let’s talk about the grading system itself, which is really quite simple. The easiest way to remember what the purity of any grade of helium is, is to simply look at the grade itself.

The first number in the grade (before the decimal point) is always equal to the number of 9’s in the purity. For example, 6.0 helium = six 9s, or 99.9999%.

The second number (after the decimal point) represents the number after the last 9. So, 4.7 grade helium = four 9s and a 7, or 99.997%.

[Need industrial grade helium? Get a free, fast quote here.](http://askzephyr.com/industrial-grade-helium/)
 

An overview of the different common grades of helium
Now that we have the grading down, let’s look into the different grades of helium and what they’re primarily used for.

Note that within each of the different grades of helium, there can be even further variations within each grade, depending on your helium supplier and your specific need. Each variation will always contain the same grade purity, but it’s the impurities, things like argon, carbon dioxide, neon, nitrogen, oxygen, and even water, that will have a different composition.

Grade 6 (6.0 helium = 99.9999% purity)
The closest to 100% pure helium, 6.0 helium is used in the manufacturing of semiconductor chips – the tiny wafers that pack the power behind your smart phones, computers, tablets, televisions, and more. It’s also used in laboratories for scientific research, laser cutting, MRI machines, and as a carrier gas in gas chromatography.

Grade 5.5 (5.5 helium = (99.9995% purity)
Like 6.0 helium, 5.5 ultra pure helium gas is typically considered “research grade,” also used in chromatography and semiconductor processing, as well as lab research, MRIs, as a shielding gas in welding, a cooling gas for fiber optics, and other industries that require a fine purity helium gas.

Grade 5 (5.0 helium = 99.999% purity)
This high purity grade helium is also widely used for gas chromatography, mass spectrometry, and specific laboratory research when higher purity gases are not necessary, as well as for weather balloons and blimps.

Grade 4.8 (4.8 helium = 99.998% purity)
The highest of the “industrial grade” heliums, 4.8 grade helium is often used by the military. The rest is classified. 😉

Grade 4.7 (4.7 helium = 99.997% purity)
A “Grade-A” industrial helium, 99.997% helium is mostly used in cryogenic applications and for pressurizing and purging, but is also used as a control atmosphere in manufacturing, as a cover gas during welding, in breathing mixtures for divers, and leak detection.

Grade 4.6 (4.6 helium = 99.996% purity)
Grade 4.6 industrial helium is used for weather balloons, blimps, in leak detection, as a shielding gas for welding, a coolant in rockets and medical applications, and as a carrier gas in the analysis of residues.

Grade 4.5 (4.5 helium = 99.995% purity)
Often the grade most commonly referred to when people say “[industrial grade](http://askzephyr.com/industrial-grade-helium/),” 99.995% helium is most commonly used in the balloon industry, but is also used as a push gas in MRI applications.

Grade 4 (4.0 helium and lower = 99.99% purity)
Any helium that is 99.99% and down into the high 80 percents is within the range of purities referred to collectively as “[balloon grade helium](http://askzephyr.com/helium-for-balloons/).” While Grade 4 helium is used mostly for balloons (although the mid-high 90 percent heliums could be used in leak detection, air bags, and heat transfer applications as well), that doesn’t necessarily mean that higher grades of helium aren’t used in balloons. In fact, in many cases, it may be Grade 5 helium.

 

Why lower grades of helium can cost more than higher grades
The majority of helium that is supplied to helium customers is actually Grade 5 helium, which is often a higher grade than is needed, particularly in the balloon industry. Despite what you may be thinking, it’s not so that helium producers can make a bigger profit. In fact, it can cost substantially more to produce a lower grade helium.

How could that be?

Phil Kornbluth’s article in this month’s CryoGas International titled, “Increased Availability of Balloon Grade Helium” does an excellent job in explaining the finer details behind the reasoning for this, but the short of it is that essentially comes down to the efficiency of mass transportation.

The majority of the world’s helium needs to be transported as a compressed, bulk liquid. That’s simply because you could only move a fraction of helium in an expanded gas state compared to a compressed liquid. (Think of recycled aluminum cans, for example. You can fit exponentially more cans in the recycling bag if you crush them down first as opposed to just tossing them in intact.)

Liquid helium is inherently extremely pure — far more pure than even Grade 5 helium actually, and liquid is the most efficient way to move product. Therefore, for helium suppliers to purposely offer a lower grade helium, they would actually have to add new operational methods and separate processes and transport specifically geared for the lower purities. All of this adds big costs of course, so most distributors simply stick to the industry standard transport of Grade 5. That is why for and end user of helium, a lower grade can cost more than the higher grades.

**Comment by Morningstar88** [comment]
https://www.youtube.com/results?search_query=call+noble+gases

**Comment by Morningstar88** [comment]
https://www.youtube.com/watch?v=rd5j8mG24H4 --- Breathing all the Noble Gases

**Comment by Morningstar88** [comment]
helium and neon are less dense than air

**Comment by Morningstar88** [comment]
argon Krypton Zenon heavier than air---https://www.airproducts.co.id/gases/helium

**Comment by Morningstar88** [comment]
Where does helium come from? Helium is a non-renewable natural resource that is most commonly recovered from natural gas deposits. Geologic conditions in Texas, Oklahoma, and Kansas make the natural gas in these areas some of the most helium-rich in the world (with concentrations between 0.3 percent and 2.7 percent).

**Comment by Morningstar88** [comment]
Toko BALON TERBANG/HELIUM
4,9
26 Google reviews 
Balloon store in Siangan, Bali
Service options: Offers same-day delivery
[Address](https://www.google.com/search?newwindow=1&sa=X&sca_esv=eba2e69e07e95a40&biw=911&bih=405&sxsrf=ACQVn09N0WUcOZpXm_hYViDg_B3aZalQ1Q:1711863116353&q=toko+balon+terbang/helium+address&ludocid=9417952147367524961&ved=2ahUKEwio7dCH452FAxVjcGwGHfIjDukQ6BN6BQifARAC): Jl. Dharma Giri, patmi, Gianyar, Kec. Gianyar, Kabupaten Gianyar, Bali 80511
[Hours](https://www.google.com/search?newwindow=1&sa=X&sca_esv=eba2e69e07e95a40&biw=911&bih=405&sxsrf=ACQVn09N0WUcOZpXm_hYViDg_B3aZalQ1Q:1711863116353&q=toko+balon+terbang/helium+hours&ludocid=9417952147367524961&ved=2ahUKEwio7dCH452FAxVjcGwGHfIjDukQ6BN6BQigARAC): 
Open 24 hours

**Comment by Morningstar88** [comment]
https://www.quora.com/Where-can-I-find-a-tank-of-pure-helium-It-s-way-too-expensive-at-the-welding-shops-hundreds-of-dollars-Thought-the-ones-used-for-balloons-were-but-it-s-now-an-80-20-blend-Thanks

**Comment by Morningstar88** [comment]
Helium is expensive because it is increasingly rare and there of course no way to economically manufacture it. Your welding shop price is typical of what you will have to pay. Why do you not want to use the blend for your balloons? I am in charge of filled balloons for a monthly birthday dinner party and the blend works quite nicely to keep our balloons floating away out there in the room air.

**Comment by Morningstar88** [comment]
Party-store helium is 20% oxygen by federal law, so that kids can’t suffocate themselves while doing chipmunk voices:

**Comment by Morningstar88** [comment]
If you need 99.99% helium, well that’s called “calibration gas” and sold by industrial supply houses like Graingers in little 103, 105-liter bottles like this:


…for about $130 plus tax.

It’s for flushing analytical gas chromatographs between samples, so it’s “clean” to well under a part per thousand minority gas constituents.

**Comment by Morningstar88** [comment]
There are some articles and videos circling the web that promote how to make your own helium for filling balloons. But can you make your own helium at home? Let’s get the facts straight.

Go do a quick search of “make your own helium” in Google and you’ll see pages upon pages of results.

(Go ahead, we’ll wait…)

Back? All of those articles and videos call for vinegar, baking soda (grade school volcano, anyone?), a bottle, a funnel, and your balloons. However, what these posts are actually showing is not how to make your own helium, but a simple method for producing carbon dioxide (CO2).

This reaction will fill the balloon as the gas escapes from the bottle, but will it float it?

Carbon dioxide is heavier than air, so the baking soda/vinegar balloon will actually fall to the ground much faster than it would if you simply blew it up with your mouth. You need a gas that is lighter than air to float the balloon, which is why we use helium.

Helium is the result of the very long, very slow decay of radioactive atoms like uranium. The alpha particles that are emitted from the decaying atom bond with loose electrons underground, producing helium atoms. Currently, this natural process is the only method with which helium is produced on Earth.

In other words: You cannot make your own helium! (Unless you happen to be a star, but stars can’t read blogs.)

**Comment by Morningstar88** [comment]
Be prepared for sticker shock. Helium ain’t cheap, and is normally sold in largeish tanks (say 10kg (weight of tank exclusive of content) and up) which you will have to buy or rent. A laboratory supply company MAY have the tiny “student” bottles (about 2kg, about the size of a 1L bottle of milk) used in high school chemistry labs and other small-scale uses.

Generally speaking, you can’t ship these in ordinary mail or courier services, because the tanks are considered hazardous materials NO MATTER WHAT IS IN THEM. You may have to pick it up yourself, or arrange the supplier to deliver it for you.

**Comment by Morningstar88** [comment]
https://www.youtube.com/results?search_query=Helium+Infused+Beer

**Comment by Morningstar88** [comment]
https://www.youtube.com/watch?v=0mu0R5evc0Y --- Two women drink HELIUM wine and the results are hilarious! - Daily Mail

**Comment by Morningstar88** [comment]
https://www.youtube.com/watch?v=IK4OyNRryIk --- https://youtu.be/IK4OyNRryIk?t=57
